src/convertToObject.js

```diff
@@ -1,18 +1,15 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';')
+    .map((styleStr) => styleStr.split(':').map(val => val.trim()))
+    .filter(val => val.length === 2)
+    .reduce((prev, next) => (
+      {
+        ...prev,
+        [next[0]]: next[1],
+      }
```

Comment:
looks good, but let''s improve solution.

map and filter loops are redundant.
try to make everything in reduce method.